algorithm_catalog/eurac_pv_farm_detection/eurac_pv_farm_detection_description.md

```diff
@@ -2,17 +2,14 @@
 
 Photovoltaic farms (PV farms) mapping is essential for establishing valid policies regarding natural resources management and clean energy. As evidenced by the recent COP28 summit, where almost 120 global leaders pledged to triple the world’s renewable energy capacity before 2030, it is crucial to make these mapping efforts scalable and reproducible. Recently, there were efforts towards the global mapping of PV farms [1], but these were limited to fixed time periods of the analyzed satellite imagery and not openly reproducible. 
 
-To resolve this limitation we implemented the detection workflow   for mapping solar farms using Sentinel-2 imagery in an openEO process [2].
+To resolve this limitation we implemented the detection workflow for mapping solar farms using Sentinel-2 imagery in an openEO process [2].
```

Comment:
this reference to `[2]` doesn't work anymore (there is only 1 item in the literature refs